akka-cluster/src/multi-jvm/scala/akka/cluster/SplitSpec.scala

```diff
@@ -0,0 +1,193 @@
+/**
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.cluster
+
+import com.typesafe.config.ConfigFactory
+import akka.remote.testkit.MultiNodeConfig
+import akka.remote.testkit.MultiNodeSpec
+import akka.remote.transport.ThrottlerTransportAdapter.Direction
+import scala.concurrent.duration._
+import akka.testkit._
+import akka.testkit.TestEvent._
+import java.util.concurrent.ThreadLocalRandom
+import akka.remote.testconductor.RoleName
+import akka.actor.Props
+import akka.actor.Actor
+import scala.util.control.NoStackTrace
+import akka.remote.QuarantinedEvent
+import akka.actor.ExtendedActorSystem
+import akka.remote.RemoteActorRefProvider
+import akka.actor.ActorRef
+import akka.dispatch.sysmsg.Failed
+import akka.actor.PoisonPill
+import akka.actor.Terminated
+
+object SplitMultiJvmSpec extends MultiNodeConfig {
```

Comment:
Yes you are right, and it does indeed fail the exact same way, not sure how I could miss that. I'll update that and delete this instead.